src/Maker/MakeDto.php

```diff
@@ -0,0 +1,250 @@
+
+        // filter id from fields
+        $fields = array_filter($fields, function ($field) {
+            // mapping includes id field when property is an id
+            if (!empty($field['id'])) {
+                return false;
+            }
+
+            return true;
+        });
+
+        $DTOClassPath = $generator->generateClass(
+            $dataClassNameDetails->getFullName(),
+            __DIR__.'/../Resources/skeleton/dto/Data.tpl.php',
+            array_merge(
+                [
+                    'fields' => $fields,
+                    'addHelpers' => $addHelpers,
+                    'omitGettersSetters' => $omitGettersSetters,
+                ],
+                $boundClassVars
+            )
+        );
+
+        $generator->writeChanges();
+        $manipulator = $this->createClassManipulator($DTOClassPath, $omitGettersSetters);
+        $mappedFields = $this->getMappedFieldsInEntity($metaData);
+
+        // Did we import assert annotations?
+        $assertionsImported = false;
+
+        foreach ($fields as $fieldName => $mapping) {
+            if (!\in_array($fieldName, $mappedFields)) {
+                continue;
+            }
+
+            $annotationReader = new AnnotationReader();
+
+            // Lookup classname for inherited properties
+            if (array_key_exists('declared', $mapping)) {
+                $fullClassName = $mapping['declared'];
+            } else {
+                $fullClassName = $boundClassDetails->getFullName();
+            }
+
+            // Property Annotations
+            $reflectionProperty = new \ReflectionProperty($fullClassName, $fieldName);
+            $propertyAnnotations = $annotationReader->getPropertyAnnotations($reflectionProperty);
+
+            $comments = [];
+
+            foreach ($propertyAnnotations as $annotation) {
+                // we want to copy the asserts, so look for their interface
+                if ($annotation instanceof Constraint) {
+                    $assertionsImported = true;
+                    $comments[] = $manipulator->buildAnnotationLine('@Assert\\'.(new \ReflectionClass($annotation))->getShortName(), (array) $annotation);
+                }
+            }
+
+            $manipulator->addEntityField($fieldName, $mapping, $comments);
+        }
+
+        $this->fileManager->dumpFile(
+            $DTOClassPath,
+            $manipulator->getSourceCode()
+        );
+
+        $this->writeSuccessMessage($io);
+
+        if (true === $assertionsImported) {
+            $io->note([
+                'The maker imported assertion annotations.',
+                'Consider removing them from the entity or make sure to keep them updated in both places.',
+            ]);
+        }
+
+        $io->text([
+            'Next: Create your form with this DTO and start using it:',
+            '$ php bin/console make:form '.$boundClassDetails->getShortName(),
```

Comment:
This is how we've written this type of message in the past: https://github.com/symfony/maker-bundle/blob/4d2743d820004f488a3ac0861ac611ca5da5b293/src/Maker/MakeMigration.php#L99